packages/form-js-viewer/src/util/index.js

```diff
@@ -143,14 +143,12 @@ export function getSchemaVariables(schema, expressionLanguage = new FeelExpressi
       const property = get(component, prop.split('.'));
 
       if (property && !expressionLanguage.isExpression(property) && templating.isTemplate(property)) {
-
         const templateVariables = templating.getVariableNames(property);
-
         variables = [ ...variables, ...templateVariables ];
       }
     });
 
-    return variables;
+    return variables.filter(variable => variable !== undefined || variable !== null);
```

Comment:
Not 100% sure about this one. The good thing about the current behavior (keeping null values) was that we catched the error in Tasklist really easily and it didn't slip our attention. However we should not crash the whole workflow, so I believe it's fine now.